File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        answer = num_list.length <= 10 ? 1 : 0;
        
        for (int i = 0; i < num_list.length; i++) {
            if (num_list.length <= 10) {
                answer *= num_list[i];
            } else {
                answer += num_list[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.py
def solution(num_list):
    
    # if len(num_list) >= 11 
    # sum
    if len(num_list) >= 11:
        return sum(num_list)
    else:
        starter = 1
        for num in num_list:
            starter *= num
        return starter
    # else 
    # multiply
    